GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -4,6 +4,62 @@
  * This project is released under the MIT License.
  */
 
+// Type Definitions
+
+/**
```

Comment:
Can we prefix all of these by `Sprite`?
SpritePointData, SpriteCustomPointData, SpriteCenterPointData, SpriteFrameData, SpriteDirectionData, SpriteAnimationData.

Sprite indicates that these types are related to sprites, and should probably not be used outside of it and will probably break one day if you do (if we had a bundle, these types would NOT be `export`ed).
Data indicate that this is part of the data used to initialize the object.

Reply:
Will separate this commit from the "small fixes"

Reply:
Yep seems like the best idea as this requires a more careful/in depth work for typing this file.

Reply:
Will do in THIS PR, as it is important in my eyes to have it always clear if you see what I mean I don't like changing a name between two commits